### ISSUE TITLE
Error deploying in CodeBuild

### ISSUE THREAD

**Issue by Mikuz** [sid88in/serverless-appsync-plugin#79]
Deployment works in my command line but in AWS CodeBuild (image aws/codebuild/nodejs:6.3.1) it breaks. Deployment in CodeBuild works fine on serverless-appsync-plugin 0.0.6 but breaks on serverless-appsync-plugin 0.0.7.

Taking a wild guess, to me it seems probable that the old node version used in CodeBuild causes this error. I find that CodeBuild should be supported and therefore the latest node version available in CodeBuild (6.3.1) should be supported.

"serverless deploy" command breaks:
```
[Container] 2018/04/12 07:38:52 Running command serverless deploy -v --stage ${BUILD_LEVEL}

Serverless Error ---------------------------------------

Serverless plugin "serverless-appsync-plugin" initialization errored: Unexpected token )
```
"serverless update-appsync" command breaks:
```
[Container] 2018/04/12 07:46:06 Running command serverless update-appsync --stage ${BUILD_LEVEL}

Serverless Error ---------------------------------------

Serverless plugin "serverless-appsync-plugin" initialization errored: Unexpected token )
```

**Comment by sid88in** [comment]
@Mikuz the deployment is not tested with AWS codebuild, but the error above initialization errored: Unexpected token ) - seems to show package mismatch (can you please install latest node and serverless packages)

**Comment by Mikuz** [comment]
My CodeBuild script (buildspec.yml) always installs serverless before it runs.
npm install -g serverless

The problem was CodeBuild's Node version. I was able to fix the problem by adding these commands to my buildspec.yml:
```
sudo npm install n -g
sudo n stable
```

The latest Node image that is currently available in CodeBuild is 6.3.1.
